dist/tools/flake8/check.sh

```diff
@@ -29,16 +29,16 @@ then
 fi
 
 ${FLAKE8_CMD} --version &> /dev/null || {
-    printf "${CERROR}$0: cannot execute \"${FLAKE8_CMD}\"!${CRESET}\n"
+    printf "%s%s: cannot execute \"%s\"!%s\n" "${CERROR}" "$0" "${FLAKE8_CMD}" "${CRESET}"
     exit 1
 }
 
-ERRORS=$(${FLAKE8_CMD} --config=${DIST_TOOLS}/flake8/flake8.cfg ${FILES})
+ERRORS=$(${FLAKE8_CMD} --config="${DIST_TOOLS}/flake8/flake8.cfg" "${FILES}")
```

Comment:
Quoting `${FILES}` here is an issue since it is a space separated list of files. Quoting it changes it to a single argument here. The proper way to solve this would be to use a bash array, but this would restrict these scripts from POSIX shell to bash as interpreter.

Reply:
> this would restrict these scripts from POSIX shell to bash as interpreter.

Is this really a problem since the script is explicitly using bash (see first line) ?

Reply:
> Is this really a problem since the script is explicitly using bash (see first line) ?

Unless somebody really wants to keep it POSIX shell compliant, no. But quoting `FILES` here is going to break things now.

Reply:
Then let's remove the quotes for now. The shellcheck script is not yet in the check scripts.

Reply:
Done